setup.py

```diff
@@ -60,8 +60,6 @@
         # this is really needed only in lint and type validations.
         # Is there any better place to put this in?
         "packaging~=21.3",
-        # Needed to fix issues with router's certificates
-        "cryptography==36.0.2",
```

Comment:
Don't we want to keep at least a light pinning, avoiding unexpected major version bump?

eg `cryptography~=39.0`

Reply:
This is also important for idempotent builds on a commit ref.

Reply:
Since this is not our direct dependency, and `pyOpenSSL` is [managing this](https://github.com/pyca/pyopenssl/blob/main/setup.py#L101), do we still need to pin it?

Reply:
Great,  did not see that, sorry. Good for me then.

Reply:
To achieve idempotent builds on a commit ref, we either need a freeze requirement.txt or use something like `poetry`